inithome/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "rubin.nublado.inithome"
```

Comment:
There isn't any problem with this name, really, but I've been using shorter names for things like this that we'll never upload to PyPI and whose output artifact is a Docker image. So following the controller, I'd just call this inithome, personally, and save the typing. But I think either is fine.

Reply:
Eh, I'd prefer to keep it inside an obviously-ours namespace.